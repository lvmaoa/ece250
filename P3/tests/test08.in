i carmen
i detectives
size
load
size
print
exit

