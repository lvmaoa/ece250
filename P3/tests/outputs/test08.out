success
success
number of words is: 2
success
number of words is: 275
